### ISSUE TITLE
Compile Error: No visible @interface for 'NSViewController' declares the selector 'viewDidAppear'

### ISSUE THREAD

**Issue by zerweck** [OpenEmu/OpenEmu#1804]
/.../OpenEmu/OpenEmu/OEFeaturedGamesViewController.m:117:12: No visible @interface for 'NSViewController' declares the selector 'viewDidAppear'

Issue happens on OS X 10.8, Macbook Air 2011, Intel HD 3000, XCode 5.0 when following the manual for compilation.

**Comment by kylewlacy** [comment]
I had this problem as well with Mavericks. The issue is that the method `[NSViewController viewDidAppear]` is only in the 10.10 SDK. To fix the issue, I did the following:

1. Download the [Xcode 6.2 beta](https://developer.apple.com/xcode/downloads/). It's only for 10.9 and up, but this shouldn't be a problem since you aren't actually installing it.
2. Open the new `.dmg`, right click on the `Xcode-Beta`, and choose "Show Package Contents"
3. Copy the directory `Xcode-Beta.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.10.sdk` into your current Xcode SDKs folder (probably `/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs`)
4. Restart Xcode, repeat the OpenEmu build instructions

Again, I only tested this in Mavericks 10.9, with Xcode 6, so YMMV.
